docs/modules/plugins/partials/azure-profile-and-subscription.adoc

```diff
@@ -6,6 +6,8 @@ could be optionally specified using either global property `azure.environment`
 `azure.{azure-service-name}.environment`. The plugin-specific property takes
```

Comment:
I propose to adjust the message in advance, for an example remove mention of `azure.{azure-service-name}.environment`

Reply:
People may use this property, thus it's. important for them to understand how it works